client/src/components/forms/Payment.js

```diff
@@ -0,0 +1,100 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import { CartContext } from "../context/Context";
+function Payment() {
+  const [cart] = useContext(CartContext);
+
+  const total = cart
+    .map((item) => {
+      return item.price * item.quantity;
+    })
+    .reduce((acc, curVal) => {
```

Comment:
I like how you're using reduce, filter etc 👏 

Though, try to avoid unnecessary abbreviations. Prefer `accumulator, currentValue` etc

Reply:
sum it is then!